packages/frontend/src/features/scheduler/components/SchedulerFilters.tsx

```diff
@@ -80,19 +86,17 @@ const FilterItem: FC<SchedulerFilterItemProps> = ({
     onRevert,
     hasChanged,
 }) => {
+    const theme = useMantineTheme();
     const { fieldsMap } = useFiltersContext();
-    const field = fieldsMap[dashboardFilter.target.fieldId];
+    const field = fieldsMap[dashboardFilter.target.fieldId] as
+        | FieldWithSuggestions
+        | undefined;
```

Comment:
Typescript doesn't consider `fieldsMap[dashboardFilter.target.fieldId]` to possibly be undefined causing us to miss this edge case.